js/app.js

```diff
@@ -164,7 +145,39 @@ function fillArray() {
   }
 }
 
-//Functions that happen on page load
+//Function to calculate the percentage of click through rate
+function clickThroughRate() {
+  for(var i = 0; i < ImageCreator.allImages.length; i++) {
+    percentageArray[i] = Math.floor(100 * (ImageCreator.allImages[i].clicks / ImageCreator.allImages[i].views));
+  }
+}
+
+//Functions that happen on page load - now including logic test for localStorage
+if (localStorage.getItem('objectArray') === null) {
```

Comment:
Just FYI: Since `null` is a lack of value, it is falsey. Thus this can also be written:
```javascript
if (!localStorage.getItem('objectArray')) {
```

Reply:
Best practice is to minimize code on the global scope, so this should probably be inside a function.